enterprise/server/remote_execution/containers/ociruntime/BUILD

```diff
@@ -102,6 +102,72 @@ go_test(
     ],
 )
 
+# keep
```

Comment:
nit: we have a few of these types of targets elsewhere but they are a bit of a hassle to maintain because gazelle doesn't handle them properly. I think it would be fine to use a flag (instead of env var) to guard the test, and then people could run it by passing the flag to the "normal" test target, like `bazel test enterprise/server/remote_execution/containers/ociruntime:ociruntime_test --config=remote --test_arg=-test_pull=true` (optionally adding `--test_filter=TestPullImage` to run just that test case)

Reply:
will fix